rgf/sklearn.py

```diff
@@ -614,6 +617,8 @@ def fit(self, X, y, sample_weight=None):
             self._estimators = [None]
             y = (y == self._classes[0]).astype(int)
             self._estimators[0] = _RGFBinaryClassifier(**params)
+            if self.n_jobs != 1:
+                print('n_jobs = {}, but RGFClassifier use one CPU because classes_ is one'.format(self.n_jobs))
```

Comment:
Default value is -1, so user will see this message very often. At least, we should print it in case of `if self.n_jobs != 1 and self.verbose`.